src/mcts/node.h

```diff
@@ -209,35 +208,49 @@ class Node {
   std::string DebugString() const;
 
  private:
-  // List of edges.
+  // Turns out the alignment and padding of the node was costing more than 8
```

Comment:
Those "turns out"-like comments are really too long for someone to read them and update when they are wrong. I don't think that any of padding-related comments are needed.
Maybe just something like: `// This class has lots of instances, take care about padding when adding new members` but even that is too obvious.

Reply:
given how dramatic an effect it can have, I'd keep an arguably too obvious comment. honestly I'm kind of surprised that compilers don't do this themselves